README.md

```diff
@@ -140,10 +140,16 @@ instead of:
 Add the following to your aliases
 
 ```clojure
-:kibit {:extra-deps {jonase/kibit {:mvn/version "0.1.8"}}
-         :main-opts ["-e" "(require,'[kibit.driver,:as,k])(k/external-run,[\"src\"],nil)"]}
+:kibit {:extra-deps {jonase/kibit {:mvn/version "0.2.0"}}
```

Comment:
I made the assumption that the next release will be `0.2.0`. Please let me know if this needs to be changed

Reply:
I don't think this is necessary. We can update all of the version references as one when it comes time to cut a new release.

Reply:
Cool. I added placeholders instead to be updated at the time of the next release.